src/Symfony/Component/Console/Tests/Helper/HelperTest.php

```diff
@@ -21,25 +21,27 @@ public static function formatTimeProvider()
     {
         return [
             [0,      '< 1 sec'],
+            [0.95,   '< 1 sec'],
             [1,      '1 sec'],
             [2,      '2 secs'],
             [59,     '59 secs'],
+            [59.21,  '59 secs'],
             [60,     '1 min'],
-            [61,     '1 min'],
-            [119,    '1 min'],
+            [61,     '1 min, 1 sec'],
+            [119,    '1 min, 59 secs'],
             [120,    '2 mins'],
-            [121,    '2 mins'],
-            [3599,   '59 mins'],
+            [121,    '2 mins, 1 sec'],
+            [3599,   '59 mins, 59 secs'],
             [3600,   '1 hr'],
-            [7199,   '1 hr'],
+            [7199,   '1 hr, 59 mins, 59 secs'],
             [7200,   '2 hrs'],
-            [7201,   '2 hrs'],
-            [86399,  '23 hrs'],
+            [7201,   '2 hrs, 1 sec'],
+            [86399,  '23 hrs, 59 mins, 59 secs'],
             [86400,  '1 day'],
-            [86401,  '1 day'],
-            [172799, '1 day'],
+            [86401,  '1 day, 1 sec'],
+            [172799, '1 day, 23 hrs, 59 mins, 59 secs'],
```

Comment:
Do we need precision to the second when the duration is in days? 2 units should be enough.

```suggestion
            [86401,  '1 day'],
            [172799, '1 day, 23 hrs'],
```

Reply:
Good idea, I'll add it.

Reply:
Is we add a `$precision = 1` to `Helper::formatTime()`, it keeps the function backward compatible and allows. To fine-tune the display by updating the placeholder function.

Reply:
@maxbeckers thanks. I see we thought of the same solution.

Reply:
Yes you're right that we that keep the current behavior, but then we don't have the benefit for the functions here like this one:
https://github.com/symfony/symfony/blob/3147082e35faadd873a801a4011bc8cbb79807e0/src/Symfony/Component/Console/Helper/ProgressBar.php#L551-L556
Because to make this a bit more exact was the issue. So i could make the default to `$precision = 1` and maybe change it here to `2` or i keep it everywhere with 2. 

Fine with both ways, what do you think?

Reply:
I prefer that you add the argument where the function is called. @wouterj What do you think?

Reply:
Changet it to default 1 and added the 2 to the function calls.